conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,202 @@
+# Jupyter Code of Conduct - Enforcement Manual
+
+This is the enforcement manual followed by Jupyter's Code of Conduct
+Committee. It's used when we respond to an issue to make sure we're consistent
+and fair.
+
+## The Code of Conduct Committee
+
+All responses to reports of conduct violations will be managed by a Code of
+Conduct Committee ("the committee"), currently comprised of [*note:* still gathering volunteers, this will be filled in as they become available and prior to final upload into the rendered site]:
+
+* Person One foo@email (Chair of the Committee)
+* ...
+* Person Five bar@email
+
+The Jupyter Steering Council will approve the members of this committee,
+comprised of five members. The members will be drawn from our entire community,
+and we intend that it *not* be only Steering Council members: anyone who is
+actively engaged with Jupyter can propose to join this committee (we'll
+explicitly solicit volunteers when short of five members).  One member will be
+designated chair of the group and will be responsible for all reports back to
+the Steering Council.
+
+
+## Enforcement guidelines and principles
+
+Enforcing the Code of Conduct impacts our community today and for the future.
+It's an action that we do not take lightly. When reviewing enforcement
+measures, the Code of Conduct Committee will keep the following values and
+guidelines in mind:
+
+* Act in a personal manner rather than impersonal.  The Committee can engage
+  the parties to understand the situation, while respecting the privacy and any
+  necessary confidentiality of reporters.  But sometimes it is necessary to communicate with one or more individuals directly: the Committee's goal is to improve the health of our community rather than only produce a formal decision.
+
+* Emphasize empathy for individuals rather than judging behavior, avoiding
+  binary labels of "good" and "bad/evil". Overt, clear-cut agression and
+  harrassment exists and will be addressed unambiguously.  But many scenarios
+  that can prove challenging to resolve are those where normal disagreements
+  devolve into inappropriate behavior from multiple parties.  Understanding the
+  full context and finding a path that re-engages all is hard, but ultimately
+  the most productive for our community.
+
+* Increase engagement in good discussion practice.
```

Comment:
I am not fully sure I understand what this means in this context. These are the principles//instructions for CoC itself. Are we saying that the CoC committee must increase their own engagement in "good discussion practice?" Or that they should encourage others to do that? As written this is too vague for the CoC to know how to act on it. I would clarify how this point is an instruction to the CoC members, remove, or move to another section.